app/src/main/java/com/cupcake/jobsfinder/domain/mapper/JobTitleMapper.kt

```diff
@@ -0,0 +1,16 @@
+package com.cupcake.jobsfinder.domain.mapper
+
+import com.cupcake.jobsfinder.data.dto.JobTitleDto
+import com.cupcake.jobsfinder.domain.model.JobTitle
+import com.cupcake.jobsfinder.ui.mapper.Mapper
+import javax.inject.Inject
+
+
+class JobTitleMapper @Inject constructor() : Mapper<JobTitleDto, JobTitle> {
```

Comment:
you can use the extension function instead of the mapper interface. Check it in create job branch

Reply:
but we convert JobTitleDto to JobTitle not the opposite and it the same way but the method in another file under name (mapper)

Reply:
for each data class you can use extension function in case if you eant to convert form it to another data class ex . ( JobDto -> JobUi )